core/HoistComponent.js

```diff
@@ -74,6 +74,11 @@ export function hoistComponent(config) {
         "The 'model' config passed to hoistComponent() is incorrectly specified: provide a spec returned by either uses() or creates()."
     );
 
+    warnIf(
+        !isMemo && isObserver,
+        'Cannot create an observer component without `memo`.  Memo is built-in to MobX observable. Component will be memoized.'
+    );
+
```

Comment:
Is the memo doc comment still relevant / useful?

> Components that are known to be unable to make effective use of memo (e.g. container components) may set this to `false`.

I never really understood what this meant, to be honest 😄  Is it referring to the really low-level comps like vbox?

Reply:
yeah -- still getting set in a large number of places -- main point is that if you are actually rendering children that *are passed to you*, you always get passed a new object references to those children, so memoization is always defeated.  Read that somewhere -- and understood it clearly at the time, but yeah its a tough model to really grok.